src/main/java/duke/Main.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.gui.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
```

Comment:
Minor nit-pick: I think the duke import should be separated from the JavaFX ones. Please do check if I am correct as I am not sure as well.

Reply:
I think you are right. Thank you for pointing it out!